### PR TITLE
docs: de-scope the IPFS Project Roadmap for 2019

### PR DESCRIPTION
**Pull request by daviddias**
//cc @ipfs/wg-captains

### CODE REVIEW

**Review by pgte — APPROVED** [review]

**Review by momack2 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -116,43 +111,21 @@ To support the continued growth of the IPFS Network, we will be testing, benchma
...
 Yes, you read the title right. The IPFS Community is going to organise the first "IPFS Conf" in the beautiful city of Lisbon. This will be an event for all the users and builders of the DWeb to come together to learn about the exciting progress that has been happening.
 - Host the 1st Conference for the IPFS Community.
 
 More details will be public soon, meanwhile, make plans to visit Lisbon in June! Subscribe for updates [here](https://github.com/ipfs/conf#ipfsconf-lisbon-2019).
 
-### 6. Everyone gets a simple way to add, share, retrieve and persist files and websites on the IPFS Network
-
-The primitives and the network are here. Now we need to create a great user experience for the DWeb by integrating it with the tools and types of services we use everyday.
-- Website Hosting on IPFS becomes simple and easy with migration paths from common sources (e.g. migrate or mirror Github Pages to IPFS).
-- The migration to Base32 CIDs is completed across all projects and IPFS Gateway.
```
Comment (momack2) [inline README.md:141]: This is still important (and inflight) - does it fit into any of the other epics or another bucket of work?

Comment (daviddias) [reply README.md:141]: Good catch! I kept it on the Web Browsers Roadmap and I thought I moved it here to another of the Epics, but seems I missed it. I'll correct. Thank you!

File: README.md [hunk 2]
```diff
@@ -116,43 +111,21 @@ To support the continued growth of the IPFS Network, we will be testing, benchma
...
-- Fully launch Peer Base toolkit to support building DApps that can run Online, Offline, and in Disjoint Networks.
-- Translate the Identity Manager RFC into code, providing a sandboxed way for Users to manage their DWeb Identity.
-- Support DApp security needs (API security, multi-tenancy, and secure broadcast updates).
-- Create a guidebook to popularize best practices to deal with unique UX challenges of the DWeb.
-- Continue to contribute to the adoption of the DWeb in the Web Platform (Brave, libdweb, IPFS companion), making it more accessible to users and Web Developers.
-
-The Dynamic Data & Capabilities WG is working full speed towards this. If you are passionate about building DApps and/or have field experience you would like to share, please come and participate in the [DDC Bi-weekly Sync](https://github.com/ipfs/dynamic-data-and-capabilities#bi-weekly-sync).
-
-### 5. The IPFS Community gets together for the 1st [IPFS Conf](https://github.com/ipfs/conf)
+### 4. The IPFS Community gets together for the 1st [IPFS Conf](https://github.com/ipfs/conf)
```
Comment (mikeal) [inline README.md]: We should probably cut this too.

**Review by Stebalien — APPROVED** [review]
:heart:

**Review by hsanjuan — APPROVED** [review]
Thanks!

File: README.md [hunk 3]
```diff
@@ -95,7 +90,7 @@ We've been working hard on improving and iterating on the IPFS Protocol to make
 - Complete Specification of the Protocol
 - Security Audits and Security Program
 - Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default
-- Bring js-ipfs and go-ipfs to v1.0.0
+- The migration to Base32 CIDs is completed across all projects and IPFS Gateway.
```
Comment (lidel) [inline README.md:93]: :heart:

File: README.md [hunk 4]
```diff
@@ -95,7 +90,7 @@ We've been working hard on improving and iterating on the IPFS Protocol to make
 - Complete Specification of the Protocol
 - Security Audits and Security Program
 - Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default
```
Comment (lidel) [inline README.md]: If we are focusing on package managers, then reproducibility of data imports is crucial and should be mentioned on the roadmap.  
What I mean by reproduciblity of data imports: the same data should produce the same CID, even if IPFS project decides to change default hash, chunker settings, unixfs representation etc. Package managers use cases will require an easy way to define all "variables" upfront (eg. via cli flags) of embedd them in data itself (https://github.com/ipfs/unixfs-v2/issues/15), to ensure old CIDs remain reproducible. 
 
```suggestion
- Go and JS IPFS enable modern IPFS data formats ([UnixFSv2](https://github.com/ipfs/unixfs-v2), [CIDv1](https://github.com/ipld/cid), raw blocks) by default and in a reproducible way
```

Ref. 
- https://wiki.debian.org/ReproducibleBuilds

**Review by lidel — APPROVED** [review]
LGTM, small suggestion inline.

**Review by eefahy — APPROVED** [review]

**Comment by daviddias** [comment]
Thank you all for your timely review ❤️

**Comment by alanshaw** [comment]
👍 LGTM

**Pull request merged**